netlify.toml

```diff
@@ -28,6 +28,12 @@
   status = 302
   force = true
 
+[[redirects]]
+  from = "/nixos/support.html"
+  to = "/learn.html"
```

Comment:
this should redirect to `/community.html`.

Reply:
OK